desktop-widgets/simplewidgets.cpp

```diff
@@ -306,10 +307,11 @@ void ShiftImageTimesDialog::buttonClicked(QAbstractButton *button)
 void ShiftImageTimesDialog::syncCameraClicked()
 {
 	QPixmap picture;
+	QStringList filters = imageExtensionFilters();
 	QStringList fileNames = QFileDialog::getOpenFileNames(this,
 							      tr("Open image file"),
 							      DiveListView::lastUsedImageDir(),
-							      tr("Image files (*.jpg *.jpeg)"));
+							      tr("Image files (%1)").arg(filters.join(" ")));
```

Comment:
I would not recommend to do this here.
Even if we can handle many other file formats on top of JPG we will only be able to extract Exif data for JPG. And beeing able to do this is essentially important for this sync camera function, isn't it?